lab-cortney/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', function() {
+  return gulp.src(__dirname + '/test/greet-test.js')
+    .pipe(mocha());
+});
+
+gulp.task('lint', function() {
+  return gulp.src([__dirname + '/*/*.js'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
+
+const watcher = gulp.watch('*/**/*.js', !'package.json', ['default', 'lint']);
```

Comment:
While your gulpfile works when you initialize your watch task this way I suggest you create your watch task as a task like this:
gulp.task('watch', () => {
  gulp.watch(pathstowatch, ['tasktorun', 'othertasktorun']);
});
That way you can specify it as a task like 'lint'.  For example your default task could call 'watch' at the end and start the watch task when you run gulp.  Additionally, when you start and define your gulp watch task this way gulp won't hang for a long time before it starts running.